### DISCUSSION TITLE
How add tables button in froala editor

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/wn-blog-plugin#52] [Q&A] [answered]
Hello,

In winter pages plugin i have button to add tables, in blog pages some buttons missing. 
![image](https://github.com/wintercms/wn-blog-plugin/assets/6842168/680a1729-3bab-4ab2-a0cf-17be42599c4d)

Is it possible to add the missing buttons here? In my case, I'm interested in the add tables button.

**Comment by LukeTowers** [comment, marked as answer]
@oim37 this isn't the froala editor, the blog plugin uses the markdown editor instead. If you would like to switch to using the Froala editor you can take a look at https://wintertricks.com/tricks/use-a-rich-editor-form-widget-as-the-winterblog-post-editor
